### PR TITLE
Release process updates

### PR DESCRIPTION
**Pull request by esethna**
Starting a WIP PR for release process updates over the next few weeks.

### CODE REVIEW

**Comment by jasonblais** [comment]
Adding a list of proposed changes:
 - Sample text for RC1 tweets on T-8: https://pre-release.mattermost.com/core/pl/tefx1ijyz7bs8mabuxmpq9f7pw

File: source/process/release-process.md [hunk 1]
```diff
@@ -191,6 +198,10 @@ The final release is cut. If an urgent and important issue needs to be addressed
       - Test the download links before and after updating the page
     - Add the download links, SHA key and md5 sum to [upgrade guide](http://docs.mattermost.com/administration/upgrade.html#version-archive)
     - Contact owners of [community installers](http://www.mattermost.org/installation/) or submit [PRs](https://github.com/tommyvn/mattermost-heroku/commit/94f7c5c0c5d7d2672fb6d62b6a560b4b5c1b5131) to update install version number
+        - For Puppet, Heroku and Ansible Playbook, post to Installers and Images channel announcing the new release. See [example](https://pre-release.mattermost.com/core/pl/5eh8fw3jaiyzzqoc6nfwfaioya).
+        - For Chef Cookbook, open a new issue to announce the new release. See [example](https://github.com/verifi-inc/mattermost/issues/2).
+        - For Yunohost, open a new pull request to update the version. See [example](https://github.com/kemenaran/mattermost_ynh/pull/11).
+        - For OpenShift, open a new pull request to update the version. See [example](https://github.com/goern/mattermost-openshift/pull/13).
```
Comment (esethna) [inline source/process/release-process.md]: Should we need to add this to the dot release checklist too?

**Comment by jasonblais** [comment]
Thanks, that makes sense

**Comment by jasonblais** [comment]
Adding WIP label back as we might do further updates after release retrospective next Thursday

**Comment by jasonblais** [comment]
just added a task for updating Twitter profile with the next release date at T-0

**Comment by lfbrock** [comment]
+1

**Pull request merged**